docs/reference-guides/monitoring-v2-configuration/receivers.md

```diff
@@ -167,50 +168,87 @@ The Teams receiver is not a native receiver and must be enabled before it can be
 
 <!-- https://github.com/idealista/prom2teams -->
 
-## SMS
+## Telegram
```

Comment:
So information about Telegram completely replaces the current info about SMS? Is the SMS receiver no longer an option for certain Rancher versions?

Reply:
My reasoning behind replacing SMS with Telegram was the lack of examples showing SMS notifications. Both Telegram and SMS notifications are delivered using the same "Alerting Driver" called [sachet](https://github.com/messagebird/sachet/) that's not directly provided/supported by Rancher.

As to your question, someone more knowledgable than me from the O&B team could help confirm that. 

@alexandreLamarre @joshmeranda could you chime in? Do you think that replacing SMS with Telegram is reasonable?

@pankajsqe for visibility, since you test this stuff regularly.

Reply:
SMS should still be supported for all versions of `rancher-alerting`. I'm in favor of switching the example to Telegram from SMS just in the sense that SMS is likely not the most used alerting system.

Reply:
I don't think it's a good idea to replace the information on SMS, instead of adding a section on Telegram in alongside the section on SMS. We might later get somebody who needs the SMS instructions and wants to know why we removed them.

Reply:
OK. Let me work on having a separate section for Telegram and keep the SMS docs as-is.

Reply:
I have added the SMS documentation back to the docs. @martyav let me know your thougts.